_product/se/tb-96aiot/ai.md

```diff
@@ -0,0 +1,70 @@
+---
+title: TB-96AIoT
+layout: ai-product
+permalink: /product/tb-96aiot/ai/
+shortname: tb-96aiot
+description: |-
+    The TB-96AIoT is a low-power, high-powered core board for the AIoT field. It is equipped with a powerful neural network processing unit (NPU) and is compatible with a variety of mainstream inference models such as caffe and tensor flow. Together with the bottom board CarrierBoard developed by Xiamen Beiqi Technology Co., Ltd., it can form a complete development board or evaluation board; the base board that can be customized according to the actual needs of the customer can directly form the industrial application board, which can meet the sweeping robot, drone, smart speaker. , automotive products, smart wear, security monitoring, AI computing modules and other areas of demand.
```

Comment:
```suggestion
    The TB-96AIoT is a low-power, high-powered core board for the AIoT field. It is equipped with a powerful neural network processing unit (NPU) and is compatible with a variety of mainstream inference models such as caffe and tensor flow. Together with the bottom board CarrierBoard developed by Xiamen Beiqi Technology Co., Ltd., it can form a complete development board or evaluation board; the base board that can be customized according to the actual needs of the customer or can directly form the industrial application board, which can meet the sweeping robot, drone, smart speaker, automotive products, smart wear, security monitoring, AI computing modules and other areas of demand.
```